quantum-adapters/kiwoom-adapter/src/kiwoom_api/api/news.py

```
@@ -0,0 +1,208 @@
+                "parameters": {
+                    "display": display,
+                    "start": start,
+                    "sort": sort
+                },
+                "total_available": api_response.get('total', 0),
+                "collected_count": len(news_articles),
+                "last_updated": datetime.now().isoformat()
+            }
+        )
+        
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)